values-es/strings.xml

```diff
@@ -1,27 +1,27 @@
             <string name="slide_android_description">Por favor selecciona la versión de Android que quieres usar para tu paquete Open GApps</string>
             <string name="slide_variant_description">Por favor selecciona la variante del paquete que quieres usar para tu paquete Open GApps</string>
-            <string name="appintro_root_explanation">La app ofrece un instalador integrado para instalar directamente el paquete descargado. Esta instalación requiere de <i>acceso root</i>. Si quieres usarlo, debes proporcionar acceso root. \nEste paso es completamente opcional.</string>
-            <string name="appintro_terms_explain">Los paquetes pre-compilados de OpenGApps.org se proveen como una cortesía por parte de OpenGApps.org y sin ningún tipo de garantía, bajo los terminos que sólo deben ser utilizados para uso personal, y no se permite la creación de mirrors públicos a excepción de OpenGApps.org. \n \n Así mismo, nótese que Open GApps no provee ninguna licencia para APKs de Google en sus paquetes. Los paquetes Open GApps son simplemente una forma conveniente de cargar APKs en su dispositivo. Es su responsabilidad obtener los permisos adecuados como por ejemplo comprando un dispositivo con licencia OHA con las Apps de Google pre-instaladas y/o adquiriendo las Apps de la Google Play Store.</string>
+            <string name="appintro_root_explanation">La app ofrece un instalador integrado para instalar directamente el paquete descargado. Esta instalación requiere de <i>acceso root</i>. Si quieres usarlo, debes proporcionar permiso al acceso root. \nEste paso es completamente opcional.</string>
+            <string name="appintro_terms_explain">Los paquetes pre-compilados de OpenGApps.org se proveen como una cortesía por parte de OpenGApps.org y sin ningún tipo de garantía, bajo los terminos que sólo deben ser utilizados para uso personal, y no se permite la creación de mirrors públicos a excepción de OpenGApps.org. \n \n Así mismo, nótese que Open GApps no provee ninguna licencia para APKs de Google en sus paquetes. Los paquetes Open GApps son simplemente una forma conveniente de cargar APKs en su dispositivo. Es su responsabilidad obtener los permisos adecuados como por ejemplo comprando un dispositivo con licencia OHA con las Apps de Google pre-instaladas y/o adquiriendo las Apps de Google Play Store.</string>
         <!--HINTS-->
             <string name="slide_arch_hint">La aplicación detecta automáticamente la arquitectura de la CPU del dispositivo</string>
             <string name="slide_android_hint">La aplicación detecta automáticamente la versión de Android del dispositivo</string>
             <string name="slide_variant_hint">La aplicación detecta automáticamente la variante instalada en dispositivo</string>
         <!--VARIOUS-->
-            <string name="detected">detectado</string>
+            <string name="detected">Detectado</string>
```

Comment:
This was on purpose without a capitalized letter (as in English) because it is shown behind other text